src/applications/personalization/profile360/sass/profile-360.scss

```diff
@@ -75,3 +75,8 @@ input.usa-only-phone {
 .no-wrap {
   white-space: nowrap;
 }
+
+.address-validation-input {
+  max-height: 125px;
```

Comment:
We need max-height? Pixel values scare me 😳

Reply:
@erikphansen found another way to accomplish this